devel/clib/Portfile

```diff
@@ -0,0 +1,39 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+PortGroup           makefile 1.0
+
+github.setup        clibs clib 2.1.13
+
+categories          devel
+license             MIT
+platforms           darwin linux
+
+homepage            https://www.clibs.org
+
+description         Package manager for the C programming language.
+
+long_description    {*}${description} Basically the lazy-man’s copy/paste \
+                    promoting smaller C utilities, also serving as a nice way \
+                    to discover these sort of libraries. The goal of clibs is \
+                    to provide stand-alone “micro” C libraries for developers \
+                    to quickly install without coupling to large frameworks. \
+                    You should use clib(1) to fetch these files for you and \
+                    check them into your repository, the end-user and \
+                    contributors should not require having clib(1) installed. \
+                    This allows clib(1) to fit into any new or existing C \
+                    workflow without friction.
+
+maintainers         {gmail.com:herby.gillot @herbygillot} \
+                    openmaintainer
+
+build.args-append   CC=${configure.cc} \
+                    CXX=${configure.cxx} \
+                    CPP=${configure.cpp}
```

Comment:
Isn't the point that the makefile portgroup handles this for you? You may need to set a makefile portgroup flag to get it to do exactly what's needed for this particular makefile since every makefile is different.

Reply:
You're right.  Was being extra paranoid re: TheRightCompiler.